spec/search.spec.js

```diff
@@ -105,8 +114,8 @@ describe('search', function() {
             animation.add(element('stub'));
             animation.add(searchFor);
 
-            expect(animation.find('/foobar')).toBe(searchFor);
-            expect(animation.findAll('/foobar')[0]).toBe(searchFor);
+            expect(animation.find('//foobar')).toBe(searchFor);
```

Comment:
No, there's one default Scene (named `""`) in any Animation now (it guarantees there's always one scene), but on import from Animatron this scene is replaced with a first scene of a project, so `/Scene 1/Layer 1` should work still.